robocop/run.py

```diff
@@ -45,9 +45,8 @@ def __init__(self, from_cli=False, config=None):
         self.root = os.getcwd()
         self.config = Config() if config is None else config
         self.from_cli = from_cli
-        if from_cli:
-            self.config.parse_opts()
-        else:
+        self.config.parse_opts()
```

Comment:
Note: doing this will use `sys.argv`, which is not what's wanted in the API (so, there should be a way for the API to pass the default arguments here).

Reply:
I've actually updated it locally later -> I'm passing from_cli instead, and if from_cli is set to False it will not load from sys.argv (and this flag is True only if we're running robocop from the cli).